util/maven/generate_poms.py

```diff
@@ -58,6 +58,24 @@ def pom_deps(label):
         'name': 'Dagger',
         'artifact': 'dagger',
     },
+    '//java/dagger:gwt': {
+        'name': 'Dagger GWT',
+        'artifact': 'dagger-gwt',
+        'manual_dependencies': '''
```

Comment:
And we could eventually add a `java_library` like `//:dagger_gwt_with_compiler` with appropriate `exports`, to be referenced by other projects using `bazelbuild/rules_gwt` (to be tested though).